src/ImageRunModal.jsx

```diff
@@ -218,13 +218,22 @@ export class ImageRunModal extends React.Component {
 
     getCreateConfig() {
         let createConfig = {};
+        // HACK: checking version is because API got broken from 1.2 -> 1.3
+        // Once we support only 1.3 podman, commit which introduces these changes can be reverted
```

Comment:
Please include which call was broken and in what way.

Super tiny nitpick: I prefer using active instead of passive voice: the API didn't "get broken", but "podman broke API". Passive voice makes it sound like it was an accident.